### ISSUE TITLE
Feature: Live View

### ISSUE THREAD

**Issue by imsnif** [imsnif/diskonaut#4]
`diskonaut` does not re-scan the hard-drive once it is done. If the files change, it does not know about it.

It would be cool if it could listen to the folders it scanned for changes and update itself. Whatever method we choose to implement it, let's make sure not to take up too many resources (CPU mostly) on the host machine.

**Comment by palfrey** [comment]
Alternately, a "update this folder" item? Maybe also if a delete fails due to something not existing that updates? I think there's a number of things that could solve a lot of the headaches of non-live without needing all the overheads of a full live view.

**Comment by imsnif** [comment]
I agree completely, @palfrey. I didn't investigate this very thoroughly, but I also think a live view will likely be quite challenging both to develop and maintain (especially cross platform). I opened this issue mostly in case someone comes up with a trivial way of doing this.

I think I was thinking along similar lines as your suggestion. My general idea was:
1. Start by displaying the last scanned time on top (https://github.com/imsnif/diskonaut/issues/3)
2. Provide a shortcut that will re-scan the current folder (and all its descendants).
3. Be smart about the re-scan (eg. prompt the user to re-scan if the last scan is X time ago, and your idea that I like about re-scanning when a deletion fails)
4. Keep a cache on the HD and use it (if it's not too old by some metric) when starting the app for fast loading.
